website/src/content/blog/biome-v1-7.md

```diff
@@ -0,0 +1,306 @@
+This is useful for checking that the files you want to commit are formatted and linted:
+
+```shell
+biome check --staged .
+```
+
+This is handy for writing your own [pre-commit script](/recipes/git-hooks/#shell-script).
+Note that, unstaged changes on a staged file are **not** ignored.
+Thus, we still recommend using a [dedicated pre-commit tool](/recipes/git-hooks/).
+
+`--changed` and `--staged` are not available on the command `biome ci` because they don't make sense in a CI environment.
+
+
+## Linter
+
+### New nursery rules
+
+Since _Biome v1.6_, we added several new rules.
+New rules are incubated in the nursery group.
+Nursery rules are subject to breaking changes.
+The new rules are:
+
+- [nursery/noConstantMathMinMaxClamp](/linter/rules/no-constant-math-min-max-clamp/)
+- [nursery/noDoneCallback](/linter/rules/no-done-callback/)
+- [nursery/noDuplicateElseIf](/linter/rules/no-duplicate-else-if/)
+- [nursery/noEvolvingAny](/linter/rules/no-evolving-any/)
+- [nursery/noFlatMapIdentity](/linter/rules/no-flat-map-identity/)
+- [nursery/noMisplacedAssertion](/linter/rules/no-misplaced-assertion/)
+
+### Promoted rules
+
+Once stable, a nursery rule is promoted to a stable group
+The following rules are promoted:
+
+- [complecity/noExcessiveNestedTestSuites](https://biomejs.dev/linter/rules/no-excessive-nested-test-suites)
+- [complexity/noUselessTernary](/linter/rules/no-useless-ternary/)
+- [correctness/useJsxKeyInIterable](/linter/rules/use-jsx-key-in-iterable/)
+- [performance/noBarrelFile](/linter/rules/no-barrel-file/)
+- [performance/noReExportAll](/linter/rules/no-re-export-all/)
+- [style/noNamespaceImport](/linter/rules/no-namespace-import/)
+- [style/useNodeAssertStrict](/linter/rules/use-node-assert-strict/)
+- [suspicious/noDuplicateTestHooks](/linter/rules/no-duplicate-test-hooks/)
+- [suspicious/noExportsInTest](/linter/rules/no-exports-in-test/)
+- [suspicious/noFocusedTests](/linter/rules/no-focused-tests/)
+- [suspicious/noSkippedTests](/linter/rules/no-skipped-tests/)
+- [suspicious/noSuspiciousSemicolonInJsx](/linter/rules/no-suspicious-semicolon-in-jsx)
+
+
+## Miscellaneous
+
+- By default, _Biome_ searches a configuration file in the working directory and in parent directories if it doesn't exist.
+  _Biome_ provides a _CLI_ option `--config-path` and an environment variable `BIOME_CONFIG_PATH` that allows which can be used to override this behavior.
+  Previously, the option and the environment variable required a directory containing a _Biome_ configuration file.
+  For example, the following command uses the _Biome_ configuration file located in `./config/`.
+
+  ```shell
+  biome format --config-path=./config/ ./src
+  ```
+
+  This was confusing for many users who are used to specifying the configuration file path directly.
+  The option and environment variable now accept a file, so the following command is valid:
```

Comment:
Saying "option and environment variable" for the third time is repetitive. I think the first time is enough, people understand we are talking about that.